packages/allure-playwright/src/helpers.ts

```diff
@@ -22,6 +22,12 @@ export class allure {
     });
   }
 
+  static description(value: string, html = false) {
```

Comment:
I don't like the suggested method signature. Maybe use `description(value: string, opts?: { html: boolean})` instead? Or even introduce separate `descriptionHtml` method. 

Btw settings regular (non-html) description will only work if reset `descriptionHtml`. Otherwise `descriptionHtml` will be rendered no matter what value is set to `description`.

PS I rather deprecate html descriptions due to possible security issues

Reply:
> PS I rather deprecate html descriptions due to possible security issues
Agree.

Then I just remove HTML from options at all

Reply:
done @baev